_posts/2020-12-31-balance-2020.md

```diff
@@ -0,0 +1,95 @@
+---
+
+The training paradigm has changed this year too, with the replacement of the in-person teaching by remote training. We described [our experience](https://galaxyproject.eu/posts/2020/09/15/ML-lessons-learned/)  and published two new preprints: [A Constructivist-based Proposal for Bioinformatics Teaching Practices During Lock-down](https://www.preprints.org/manuscript/202008.0532/v2) and [Fostering Accessible Online Education Using Galaxy as an e-learning Platform](https://www.preprints.org/manuscript/202009.0457/v2). 
+
+[Training Infrastructure as a Service](https://www.biorxiv.org/content/10.1101/2020.08.23.263509v1) has increased this year too, with more than 5,000 students trained since mid-2018 and more than 170 events organised. We received feedback from some training organisers: [Denny Popp on Metagenomics](https://galaxyproject.eu/posts/2020/01/20/tiaas-feedback-popp/), [Beata Scholtz on RNA-seq data analysis](https://galaxyproject.eu/posts/2020/02/10/tiaas-feedback-Scholtz/), Helena Rasche described the experience with the [Galaxy Admin Training](https://galaxyproject.eu/posts/2020/03/06/admin-training-success/), Marcel Schulz taught [RNA-seq and different kinds of epigenomics data, including ChIP-seq and ATAC-seq data](https://galaxyproject.eu/posts/2020/06/08/tiaas-feedback-schulz/), Rodrigo Ortega Polo gave an [introductory workshop to bacterial whole-genome sequence analysis](https://galaxyproject.eu/posts/2020/07/25/tiaas_feedback_rodrigo/), Leighton Pritchard gave a workshop on [genome sequencing, assembly, and comparative genomics](https://galaxyproject.eu/posts/2020/10/14/tiaas_feedback_Leighton/) and by the end of the year, Matthias Fahrner and Melanie Föll participated in the ELIXIR Belgium workshop teaching [data-dependent acquisition (DDA) and data-independent acquisition method (DIA)](https://galaxyproject.eu/posts/2020/12/08/tiaas_feedback_Melanie-Matthias/).
+
+<br>
+
+## Citizen Science
+---
+
+A crowdsourcing [citizen science project](https://galaxyproject.org/blog/2020-01-galaxy-ecology-citizen-science/) has been running since the beginning of the year to help ecologists with the classification of bees.
+
+The [streetscience community](https://streetscience.community/) has organised periodic calls all over the year. In February, [the BeerDEcoded project was introduced to scientists](https://streetscience.community/events/2020-02-19-introduction_StreetScienceCommunity/) at the beginning of a Galaxy workshop. Later in the year, [the community brewed 8.5 liters of beer!](https://streetscience.community/events/2020-09-27-BeerBrewing/)
+
+<br>
+
+## Citations
+---
+
+In 2020, the global Galaxy project reached a total of [10,000 publications](https://galaxyproject.org/blog/2020-08-10k-pubs/). Only during the year 2020, [48 publications cited the European Galaxy Server](https://galaxyproject.eu/citations). Thank you for helping us to keep this service for free by [citing it](https://galaxyproject.eu/about)!
+
```

Comment:
maybe a link to the 5 years report as well for people that want more information?